src/Symfony/Component/DependencyInjection/Tests/Fixtures/php/services13.php

```diff
@@ -30,6 +30,9 @@ public function __construct()
         $this->methodMap = array(
             'bar' => 'getBarService',
         );
+        $this->privates = array(
+            'service_container' => true,
```

Comment:
should we actually dump it in this array ? It receives a special treatment when getting it anyway